web/main/views.py

```diff
@@ -1702,13 +1702,13 @@ def create_draft(request, casebook):
 @require_http_methods(["GET", "POST"])
 @requires_csrf_token
 @hydrate_params
-def edit_casebook(request, casebook):
+def edit_casebook(request, casebook: Casebook):
     """
     Given:
     >>> private, with_draft, for_verified_prof, client = [getfixture(f) for f in ['full_private_casebook', 'full_casebook_with_draft', 'full_private_casebook_for_verified_prof', 'client']]
     >>> draft = with_draft.draft
 
-    Users can edit their unpublished and draft casebooks:
+    Users can edit their un'apublished and draft casebooks:
```

Comment:
```suggestion
    Users can edit their unpublished and draft casebooks:
```

Reply:
Thanks for the catch, errant vim keystrokes 😅 

Frontend now checks for the feature flag

Addressed in [d48ded4](https://github.com/harvard-lil/h2o/pull/1700/commits/d48ded4bbefb3f94624f711299741197a3078d08).